user-guide-chapters/zb/zb.adoc

```diff
@@ -1,17 +1,138 @@
-== What is zb?
+== Benchmarking {zotLowerName} with zb
```

Comment:
same comment as zli except this should be after the server configuration chapter